src/backend/main.ts

```diff
@@ -944,6 +946,13 @@ ipcMain.handle(
       mainWindow?.hide()
     }
 
+    if (autoCloseLauncher) {
+      mainWindow?.hide()
+      setTimeout(() => {
```

Comment:
Timeout here is okay, but what if the game crashes beforehand? How about making window pop up again if game crashed within that 8 second wait time